extensions/beancount-mate/src/components/ErrorView.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { List } from "@raycast/api";
+
+const ErrorView: React.FC = () => {
+  return (
+    <List.EmptyView
+      icon={"https://cdn-icons-png.flaticon.com/512/4160/4160755.png"}
```

Comment:
Could we save this in the extension instead and in 128x128 size - remember credits to the author/service if needed